docs/source/getting-started/howdoesitwork.md

```diff
@@ -0,0 +1,5 @@
+# How does Volto works under the hood
+
+You can watch the talk during the World Plone Day 2021:
```

Comment:
Shouldn't we add a plain link to the YT video to work w/o JS or to watch it externally?

Reply:
Maybe the embed is doing that already 🤔

Reply:
I wanted to explain a bit more with narrative, but not in the mood today and left it for the near future :P 

I can turn it into a plain link.

Reply:
I like the embed, I was wondering on an additional link for users with js disabled or for a11y.
Like a link on `watch the talk`.

But yes, not blocking for sure

Reply:
Although it renders fine...

<img width="1249" alt="image" src="https://user-images.githubusercontent.com/486927/118407964-8c8db400-b683-11eb-9f8e-dddf2658d6a7.png">

Reply:
All right, there's a link 👍

Reply:
yep!